webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
```

Comment:
I think this looks fine, I am going to modify this file in ch-docker branch by adding
`devServer: {
 port: 8080,
 host: localhost
}`

to recognise the running port for docker